Makefile

```
@@ -90,6 +92,17 @@ remote-garden-down:
 	# Remove docker containers and networks
 	@-./hack/local-development/remote-garden/cleanup $(REMOTE_GARDEN_LABEL)
 
+.PHONY: start-all-servers
+start-all-servers:
```

```suggestion
.PHONY: start-all
start-all
```

This target starts also some controllers and they are not acting as servers, but just clients.